settings.py

```diff
@@ -1,14 +1,21 @@
 
 import pygame
 from gi.repository import Gtk
+from gi.repository import Gdk
 from button import Button
 
 
+def color_parse(color):
+    rgba = Gdk.RGBA()
+    rgba.parse(color)
+    return (int(rgba.red * 255), int(rgba.green * 255), int(rgba.blue * 255))

```

Comment:
If you're using `Gdk.RGBA`, there's no need to multiply by 255.

Reply:
The background is getting black without this multiplication

Reply:
This is what I see without multiplying.

![Screenshot from 2023-07-20 23-49-03](https://github.com/sugarlabs/make-them-fall-activity/assets/16203834/9e04a04a-4caf-45ee-9617-e4f10097317a)

Reply:
sorry i didn't specify the screen, I was talking about the settings screen

Reply:
True, seems RGBA has issues with handling some colours.

Reply:
so we can let it be like that then?

Reply:
Yes we can.